### ISSUE TITLE
Write examples with larger arrays

### ISSUE THREAD

**Issue by riking** [Cryptjar/avr-progmem-rs#13]
All of the array sizes in the documentation examples seem to be fairly small: the element type is always bytes, and the size is never more than about 5. I think that an example of hash function round constants would be more compelling, such as the data found here:

https://github.com/jedisct1/rust-hmac-sha256/blob/e787a2e40ce00c7e27b336207cdec730226ff6f1/src/lib.rs#L127-L138

Here, the program needs 64 u32s and loads (up to) 16 at a time.

Motivated via <https://github.com/rust-lang/rust/issues/109000>.

**Comment by Cryptjar** [comment]
Certainly, a more elaborate example would be a welcome addition. Tho, an entire HMAC implementation seems a bit excessive.
If you have a good idea for an example with a more 'practical' array demonstration that isn't worth an entire crate on its own, I would gladly accept your PR.

**Comment by n-eq** [comment]
Hello. for what it's worth, [here](https://github.com/n-eq/pcd8544-rs/blob/use-progmem/examples/arduino-uno-pcd8544/src/main.rs#L10) is a "real-world" example of using the library.
`progmem`  is used to store a 504 bytes bitmap that is drawn to an 84x48 LCD screen.
